src/loadTable.js

```diff
@@ -0,0 +1,23 @@
+const scoreData = [
+  { name: 'name', score: 100 },
+  { name: 'name', score: 53 },
+  { name: 'name', score: 67 },
+  { name: 'name', score: 51 },
+  { name: 'name', score: 23 },
+  { name: 'name', score: 98 },
+];
+
+function loadTable() {
+  const table = document.querySelector('.scoreboard-table');
+  scoreData.forEach((data) => {
+    const tr = document.createElement('tr');
+    tr.className = 'table-row';
+    const td = document.createElement('td');
+    td.className = 'table-data';
+    td.innerHTML = `${data.name}: ${data.score}`;
+
+    tr.appendChild(td);
+    table.appendChild(tr);
+  });
+}
```

Comment:
- [ ] ES6 syntax must be used for this project, please change the function keyword for an arrow function structure to comply with the project requirements.

Reply:
Thank you for the corrections